includes/agentic-commerce/class-wc-stripe-agentic-checkout-session.php

```
@@ -0,0 +1,253 @@
+	 * @return WC_Stripe_Agentic_Line_Item[]
+	 */
+	public function get_line_items(): array {
+		$raw_items = $this->session->line_items->data ?? [];
+
+		return array_map(
+			function ( $item ) {
+				return new WC_Stripe_Agentic_Line_Item( $item );
+			},
+			$raw_items
+		);
+	}
```

PHPMD (2.15.0)

[error] The method get_line_items is not named in camelCase. (undefined)

(CamelCaseMethodName)